bin/http-server

```diff
@@ -29,7 +29,7 @@ if (argv.h || argv.help) {
     '                     Optionally provide CORS headers list separated by commas',
     '  -o [path]    Open browser window after starting the server',
     '  -c           Cache time (max-age) in seconds [3600], e.g. -c10 for 10 seconds.',
-    '               To disable caching, use -c-1.',
+    '               To disable caching, use -c-1. Also supports a cache function. https://github.com/jfhbrook/node-ecstatic#optscache',
```

Comment:
We should get this added to the README too

Reply:
Updated README.